### PR TITLE
docs: Add bid creation example applications

### PR DESCRIPTION
**Pull request by lfportal**
### Hi👋, please prefix this PR's title with:

- [ ] `breaking-change:` if you have introduced modification that necessitates immediate adjustments by this SDK's users to their applications, clients, or integrations to avert disruptions to existing features or functionalities.
- [x] `feat:`, `fix:`, `refactor:`, `docs:`, or `chore:`.

# Summary

Add example applications for orderbook bids feature.

### CODE REVIEW

File: examples/orderbook/create-bid-with-nextjs/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "test": "playwright test"
+  },
+  "dependencies": {
+    "@biom3/react": "^0.26.1",
+    "@imtbl/sdk": "1.53.0-alpha.11",
```
Comment (lfportal) [inline examples/orderbook/create-bid-with-nextjs/package.json:14]: **TODO:** Return to `@latest` before merging.

File: examples/orderbook/fulfill-bid-with-nextjs/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "test": "playwright test"
+  },
+  "dependencies": {
+    "@biom3/react": "^0.26.1",
+    "@imtbl/sdk": "1.53.0-alpha.11",
```
Comment (lfportal) [inline examples/orderbook/fulfill-bid-with-nextjs/package.json:14]: **TODO:** Return to `@latest` before merging.

**Review by naveen-imtb — APPROVED** [review]

File: examples/orderbook/create-bid-with-nextjs/app/utils/bid.ts [hunk 3]
```diff
@@ -0,0 +1,75 @@
+import { orderbook } from "@imtbl/sdk";
+import { Web3Provider } from "@ethersproject/providers";
```
Comment (zaidarain1) [inline examples/orderbook/create-bid-with-nextjs/app/utils/bid.ts:2]: `@ethersproject/providers` is missing from the dependencies object in the package.json of this example. Could that please be added there?

File: examples/orderbook/create-bid-with-nextjs/playwright.config.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import { defineConfig, devices } from "@playwright/test";
+
+export default defineConfig({
+    testDir: "./tests",
+    fullyParallel: true,
+    forbidOnly: !!process.env.CI,
+    retries: process.env.CI ? 2 : 0,
+    workers: process.env.CI ? 1 : undefined,
```
Comment (zaidarain1) [inline examples/orderbook/create-bid-with-nextjs/playwright.config.ts:8]: The playwright config base used for examples was recently updated to work a bit better on CI. The changes made were:
`workers: "80%"`,
and
`webServer.command: "yarn start"`

Could those changes be applied to this PR?

File: examples/orderbook/fulfill-bid-with-nextjs/playwright.config.ts [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import { defineConfig, devices } from "@playwright/test";
+
+export default defineConfig({
+    testDir: "./tests",
+    fullyParallel: true,
+    forbidOnly: !!process.env.CI,
+    retries: process.env.CI ? 2 : 0,
+    workers: process.env.CI ? 1 : undefined,
```
Comment (zaidarain1) [inline examples/orderbook/fulfill-bid-with-nextjs/playwright.config.ts:8]: As mentioned earlier regarding `workers` and `webServer.command`

**Review by zaidarain1 — CHANGES_REQUESTED** [review]

**Review by zaidarain1 — DISMISSED** [review]

**Review by zaidarain1 — APPROVED** [review]

**Pull request merged**